lib/aytests/vm.rb

```diff
@@ -0,0 +1,37 @@
+require "forwardable"
+
+module AYTests
+  class VM
+    # This class allows to query and modify virtual machines attributes.
```

Comment:
Shouldn't this comment be one line before, so it's used by yardoc to describe the class?